docs/reference/openfeature-operator/crds/featureflagsource.md

```diff
@@ -63,14 +63,14 @@ sources:
 `FeatureFlagSource` further allows to provide configurations to the injected flagd sidecar.
 Table given below is non-exhaustive list of overriding options,
 
-| Configuration | Explanation                   | Default                    |
-|---------------|-------------------------------|----------------------------|
-| port          | Flag evaluation endpoint port | 8013                       |
-| metricsPort   | Metrics port                  | 8014                       |
-| evaluator     | Evaluator to use              | json                       |
-| image         | flagD image                   | ghcr.io/open-feature/flagd |
-| tag           | flagD image tag               | Latest tag                 |
-| probesEnabled | Enable/Disable health probes  | true                       |
+| Configuration  | Explanation                   | Default                    |
+|----------------|-------------------------------|----------------------------|
+| port           | Flag evaluation endpoint port | 8013                       |
+| managementPort | Management port               | 8014                       |
```

Comment:
We'll need to make sure this is updated in OFO. FYI @toddbaert @Kavindu-Dodan @thisthat

Reply:
Yes, we need a follow-up issue to migrate OFO.

However, current configurations won't break after this fix as @craigpastro did a good job of having a fallback for the deprecated option [^1]. 

[^1]: https://github.com/open-feature/flagd/blob/62033448903b20eb992014e25a365cafe955400e/flagd-proxy/cmd/start.go#L75-L80